### DISCUSSION TITLE
Build 22621.2134 is not in dropdown of latest Vivetool-GUI

### DISCUSSION THREAD

**Discussion by Globespy** [PeterStrick/ViVeTool-GUI#68] [General]
My Insider build is not listed, in fact even the beta version (22631.xxxx) of this build is not listed?
What am I doing wrong?

**Comment by PeterStrick** [comment]
> My Insider build is not listed, in fact even the beta version (22631.xxxx) of this build is not listed? What am I doing wrong?

You are doing nothing wrong. The latest Version of 22621 as a Feature List is 22621.898

![image](https://github.com/PeterStrick/ViVeTool-GUI/assets/60312421/24d3e47b-a1c8-490c-9498-37093d878222)

It may take a while for new Feature Lists to get uploaded to the mach2 Repository, from where the Combo Box populates it's entries from.

**Reply by Globespy** [reply]
thanks, so I just wait and while and try the tool again?

**Reply by PeterStrick** [reply]
Most likely, in the meantime you can enable and disable IDs that you find on the Internet by pressing F12 inside ViVeTool GUI

**Comment by PeterStrick** [comment]
> thanks, so I just wait and while and try the tool again?

**Comment by Globespy** [comment]
I'm really just trying to enable the RGB Lighting control which should be in this build, but disabled by default on the Insider RP channel, but enabled on the beta channel build.
The id apparently is 35262205
